### PR TITLE
Elementary setup blog

### PR DESCRIPTION
**Pull request by tbridges-ippon**

### CODE REVIEW

**Review by pkrish123 — CHANGES_REQUESTED** [review]
I have one wording change, but other than that it's good to go!

File: posts/dbt-elementary-snowflake.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+authors:
+  - Tyler Bridges
+tags:
+  - DBT
```
Comment (nnightingale14) [inline posts/dbt-elementary-snowflake.md:5]:
```suggestion
  - dbt
```

File: posts/dbt-elementary-snowflake.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  - Snowflake
+  - Elementary
+  - Alerting
+date: 2023-06-15T12:00:00.000Z
+title: "Enhance your DBT Data Monitoring with Elementary"
```
Comment (nnightingale14) [inline posts/dbt-elementary-snowflake.md:10]:
```suggestion
title: "Enhance your dbt Data Monitoring with Elementary"
```

File: posts/dbt-elementary-snowflake.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+---
+
+# Enhance your DBT Data Monitoring with Elementary
+
+<p> This blog aims to provide a guide on implementing the Elementary package with DBT. The primary functionalities of this guide focus on: </p>
```
Comment (nnightingale14) [inline posts/dbt-elementary-snowflake.md:15]:
```suggestion
<p> This blog aims to provide a guide on implementing the Elementary package with dbt. The primary functionalities of this guide focus on: </p>
```

**Review by nnightingale14 — APPROVED** [review]

File: posts/dbt-elementary-snowflake.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+Elementary's functionality revolves around its dbt package, which enables data monitoring and collection of dbt artifacts. Configuration for monitoring is done within your dbt project, using dbt macros and models as monitors. The collected data is stored in a dedicated schema called "elementary" in your data warehouse. To generate UI reports and send Slack alerts, Elementary CLI comes into play. This command-line interface simplifies the process of creating reports and ensures timely notifications of data issues. With Elementary, you can effortlessly monitor your dbt projects, detect anomalies, and gain valuable insights into your data's health and performance, enabling you to proactively address issues and optimize your data workflows.
+
```
Comment (portillosc) [inline posts/dbt-elementary-snowflake.md:26]: ^ the two previous paragraphs are concise and well worded.

**Review by pkrish123 — APPROVED** [review]

**Pull request merged**